src/geom/bounding_box.C

```diff
@@ -62,6 +62,62 @@ bool BoundingBox::contains_point (const Point & p) const
 }
 
 
+
+Real BoundingBox::max_size() const
+{
+  Real size = 0;
+  for (unsigned int d = 0; d != LIBMESH_DIM; ++d)
+    {
+      Real sized = this->second(d) - this->first(d);
+      if (!libmesh_isinf(sized))
```

Comment:
This would return `false` if `sized` corresponded to `std::numeric_limits<Real>::max()`, wouldn't it? I just read https://en.cppreference.com/w/cpp/numeric/math/isinf to check. If I'm correct, then the behavior of this function doesn't match its doxygen

Reply:
Yeah, I think you're right.  Pushed a fix.